modules/apps/29-fee/types/fee.go

```diff
@@ -7,13 +7,31 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-// NewIdentifiedPacketFees creates and returns a new IdentifiedPacketFees struct
-func NewIdentifiedPacketFees(packetFees []IdentifiedPacketFee) IdentifiedPacketFees {
```

Comment:
what happened to this constructor?

Reply:
Replaced by `NewPacketFees()` on line 20 below. 

The `IdentifiedPacketFees` struct used for Genesis is a repeated type so the `[]IdentifiedPacketFees` is returned directly from `GetAllIdentifiedPacketFees` and used in `ExportGenesis`. See #957 -> here https://github.com/cosmos/ibc-go/pull/957/files#diff-b7d50f2bf0988eeab4d6734bd38c10ebf00bda7451e4a6730575de154406937cR330